tests/wpt/mozilla/meta/MANIFEST.json

```diff
@@ -7845,6 +7845,11 @@
    ]
   },
   "support": {
+   "./.DS_Store": [
+    [
+     {}
+    ]
+   ],
```

Comment:
I think I'll have to manually remove these ./.DS_Store part

Reply:
You should just be able to revert all of the MANIFEST.json changes entirely.